server/config/config.go

```diff
@@ -735,6 +735,12 @@ type ScheduleConfig struct {
 	// is overwritten, the value is fixed until it is deleted.
 	// Default: manual
 	StoreLimitMode string `toml:"store-limit-mode" json:"store-limit-mode"`
+
+	// Controls the time interval between write hot regions info into leveldb.
+	HotRegionsWriteInterval typeutil.Duration `toml:"hot-regions-write-interval" json:"hot-regions-write-interval"`
+
+	// The day of  hot regions data to be reserved.0 means close
+	HotRegionsResevervedDays int64 `toml:"hot-regions-reserved-days" json:"hot-regions-reserved-days"`
```

Comment:
Why not use `typeutil.Duration`?

Reply:
The unit of this is days, does toml support days?

Reply:
So, the minimum time of reserving the hot regions is 1 day?

Reply:
yes,the minimum time of reserving the hot regions is 1 day

Reply:
What if we just want to keep the history for several hours?

Reply:
we delete history hot region every 24 hours,if we want reserve several hours,we must change delete function